src/lib/chat/index.ts

```diff
@@ -138,6 +139,10 @@ export class Chat {
     return this.client.fetchConversationsWithUsers(users);
   }
 
+  async markRoomAsRead(roomId: string, userId?: string): Promise<void> {
+    return this.client.markRoomAsRead(roomId, userId);
```

Comment:
if we're not `awaiting` then we can remove `async` from this function

Reply:
Yeah, `async` is required here